doc/configuration/configuration.rst

```diff
@@ -0,0 +1,12 @@
+.. _Configuration_details:
+
+#######################
+GIS Configuration
+#######################
+
+
+The CSC for the GIS is configurable in a few ways.
+
+1. Host: The IP address of the modbus server for communicating with the GIS
+2. Port: The port that the server is running on
+3. telemetry_interval: The interval of time that the telemetry loop is updated.
```

Comment:
Why are Host and Port written with capital initial letters and not telemetry_interval? Please stick to one of them.